backend/canisters/local_user_index/impl/src/lib.rs

```diff
@@ -215,6 +218,10 @@ struct Data {
     pub rng_seed: [u8; 32],
 }
 
+fn identity_canister_id() -> CanisterId {
+    CanisterId::from_text("6klfq-niaaa-aaaar-qadbq-cai").unwrap()
```

Comment:
Use test flag to set this to prod_test id

Reply:
The test flag isn't easily available at this stage.
I can just upgrade it first on prod test with the value set to the prod test canisterId.